openssl/src/pkey.rs

```diff
@@ -250,6 +250,34 @@ where
     {
         unsafe { ffi::EVP_PKEY_cmp(self.as_ptr(), other.as_ptr()) == 1 }
     }
+
+    /// Raw byte representation of a public key
+    ///
+    /// This corresponds to [`EVP_PKEY_get_raw_public_key`].
+    ///
+    /// [`EVP_PKEY_get_raw_public_key`]: https://www.openssl.org/docs/man1.1.1/man3/EVP_PKEY_get_raw_public_key.html
+    ///
+    /// This function only works for algorithms that support raw public keys.
+    /// Currently this is: X25519, ED25519, X448 or ED448
+    #[cfg(ossl111)]
+    pub fn public_key_to_raw_bytes(&self) -> Result<Vec<u8>, ErrorStack> {
```

Comment:
I would just call these `raw_public_key` and `raw_private_key` to match the function names.

Reply:
Sure, no problem. Are those tests failing due to something in my code changes? Or is it a problem with the CI?

Reply:
You need to add the raw FFI bindings to openssl-sys.

Reply:
If the methods are only available on 1.1.1, you need to add cfgs to the tests as well.

Reply:
Just pushed an update that should address everything